### DISCUSSION TITLE
스터디 12주차 문제

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#184] [공지사항]
> [!NOTE]
> 풀이 기간: 2024년 7/14(일)~7/20(토)

## Longest Increasing Subsequence
- 문제: https://leetcode.com/problems/longest-increasing-subsequence/
- 풀이: https://www.algodale.com/problems/longest-increasing-subsequence/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/longest-increasing-subsequence

## Unique Paths
- 문제: https://leetcode.com/problems/unique-paths/
- 풀이: https://www.algodale.com/problems/unique-paths/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/unique-paths

## Longest Common Subsequence
- 문제: https://leetcode.com/problems/longest-common-subsequence/
- 풀이: https://www.algodale.com/problems/longest-common-subsequence/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/longest-common-subsequence

## Maximum Subarray
- 문제: https://leetcode.com/problems/maximum-subarray/
- 풀이: https://www.algodale.com/problems/maximum-subarray/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/maximum-subarray

## Jump Game
- 문제: https://leetcode.com/problems/jump-game/
- 풀이: https://www.algodale.com/problems/jump-game/
- 위치: https://github.com/DaleStudy/leetcode-study/tree/main/jump-game

**Comment by dev-jonghoonpark** [comment]
> Jump Game
> 문제: https://leetcode.com/problems//jump-game/
> 풀이: https://www.algodale.com/problems//jump-game/

`/` 가 두 개씩 들어가있어서 수정해두겠습니다...!

@DaleSeo
